Tasks/PublishTestResultsV2/publishtestresults.ts

```diff
@@ -1,12 +1,10 @@
-import * as fs from 'fs';
-import * as os from 'os';
 import * as path from 'path';
 import * as publishTestResultsTool from './publishtestresultstool';
 import * as tl from 'vsts-task-lib/task';
-import * as tr from 'vsts-task-lib/toolrunner';
 import { publishEvent } from './cieventlogger';
 
 const MERGE_THRESHOLD = 100;
+const TESTRUN_SYSTEM = 'PublishTestResults';
```

Comment:
Should this be "VSTS - publish" as suggested by Vinod before?

Reply:
I agree it should be VSTS - PTR